README.md

```diff
@@ -15,6 +15,18 @@ scripts in `bin/`. `warmup_stats` takes either CSV files or
 [Krun](http://soft-dev.org/src/krun/) results files as input. As output it can
 create HTML or LaTeX / PDF tables and diffs, or PDF plots.
 
+## Terminology
+
+warmup_stats uses the following terminology:
+
+  * A *process execution* is a single run of a single benchmark in a single
```

Comment:
I think this needs to be clearer. A "single run of a single benchmark" initially sounds like an IPI. Then later we describe a process execution in the section for IPIs.

How about:

> An in-process iteration (IPI) is a single run of a benchmark.

> A process execution runs one or more IPIs within one operating system process. The resulting measurements form time-series data.

Reply:
Hmm, TBH, I'm not sure that makes things better -- though I agree that my attempt is not exactly ideal.

How about "A process execution involves running a program in a process from creation to termination. In other words, it is equivalent to running a program from the command-line and waiting for it to finish" and "An in-process iteration is a single run of a benchmark in a `for` loop inside a process execution"? That might still be a bit ambiguous, but I think it's a bit clearer?

Reply:
It's better.

For contrast, we managed something much more concise than both our suggestions in the warmup paper:
> We present a carefully designed experiment where benchmarks are run for 2000 in-process iterations and repeated using 30 fresh process executions (i.e. each process execution runs multiple in-process iterations).

Reply:
How about:

> A process execution is an operating system process used for benchmarking. Each process execution runs many in-process iterations using a loop.

Reply:
OK, I've taken (hopefully) the best bits and put them into https://github.com/softdevteam/warmup_stats/pull/104/commits/56ce30dc18a611889510786c9eec0a8d4aa7a43d -- see what you think. @farquet may also have a useful opinion on this!

Reply:
I like the version from 56ce30dc18a611889510786c9eec0a8d4aa7a43d. It's short and precise.